From d65df4069ef124cbb38e97960be01f79bbdac5ae Mon Sep 17 00:00:00 2001
From: Adrian Bunk <bunk@debian.org>
Date: Sun, 2 Aug 2020 15:34:41 +0300
Subject: Gia64-test-nat.c: #include <signal.h> for sigaction()

---
 tests/Gia64-test-nat.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/Gia64-test-nat.c b/tests/Gia64-test-nat.c
index 89df54e0..abd6ffa7 100644
--- a/tests/Gia64-test-nat.c
+++ b/tests/Gia64-test-nat.c
@@ -26,6 +26,7 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.  */
 /* This file tests corner-cases of NaT-bit handling.  */
 
 #include <errno.h>
+#include <signal.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-- 
2.20.1

